commitrmnocb/metrics.go

```diff
@@ -0,0 +1 @@
+package commitrmnocb
```

Comment:
Is the intention of this file to house some prometheus metrics?

Reply:
Yes